scripts/make_credentials.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+C4GH_VERSION="1.7.3"
+
+apt-get -o DPkg::Lock::Timeout=60 update > /dev/null
+apt-get -o DPkg::Lock::Timeout=60 install -y curl postgresql-client openssl > /dev/null
+
+for n in download finalize inbox ingest mapper sync verify; do
+    echo "creating credentials for: $n"
+    ## password and permissions for MQ
+    curl -s -u test:test -X PUT "http://rabbitmq:15672/api/users/$n" -H "content-type:application/json" -d '{"password": "'"$n"'", "tags":"none"}'
+    curl -s -u test:test -X PUT "http://rabbitmq:15672/api/permissions/gdi/$n" -H "content-type:application/json" -d '{"configure":"","write":"sda","read":".*"}'
+
+    ## password and permissions for DB
+    if [ "$n" = inbox ]; then
+        psql -U postgres -h postgres -d lega -c "CREATE ROLE inbox;"
+        psql -U postgres -h postgres -d lega -c "GRANT base, ingest TO inbox;"
+    fi
+
+    if [ "$n" = ingest ]; then
+        psql -U postgres -h postgres -d lega -c "GRANT UPDATE ON local_ega.main TO ingest;"
+    fi
+
+    psql -U postgres -h postgres -d lega -c "ALTER ROLE $n LOGIN PASSWORD '$n';"
```

Comment:
No, use `echo "ALTER ROLE :'name' LOGIN PASSWORD :'name'" | psql --variable=name="$n" ...` instead.

Reply:
`-c` means log in an d then run the following command, like you would do it manually.

Reply:
Yes, but you inject a shell variable into the SQL expression. 

I'll leave it as it is, but it's currently providing a way of doing an SQL injection.